.gitignore

```diff
@@ -80,6 +80,7 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+*.pxi
```

Comment:
> we DON't want to include an auto-generated file in git

we did actually include the generated code in git before (but it's good we no do not include it anymore)

> So yes maybe should specifically exclude them in .gitignore (or rather the auto-gen ones) 

that's what I meant yes

and as I also said, not that important at this moment, as we do not have any other `.pxi` files